projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -153,7 +153,8 @@ public function initial_state() {
 		$shares                  = $publicize->get_publicize_shares_info( Jetpack_Options::get_option( 'id' ) );
 		$refresh_plan_from_wpcom = true;
 		$show_nudge              = ! Current_Plan::supports( 'social-shares-1000', $refresh_plan_from_wpcom );
-
+		$protocols               = array( 'http://', 'http://www.', 'www.' );
```

Comment:
I think `https://` should be here as well, I have problems using it with a https site.

The wordpress checkout site with the plan is not loading up properly. The link would go to: https://jetpack.com/redirect/?source=jetpack-social-basic-plan-block-editor&site=https%3A%2F%2Fgmjuhasztest.jurassic.tube&query=redirect_to%3Dhttps%3A%2F%2Fgmjuhasztest.jurassic.tube%2Fwp-admin%2Fpost-new.php

If I change the site in the params, and remove the `https://` it works. Can't we get the name of the site somehow, so we dont have to bother with all the protocols?

Reply:
Thanks Geri! I have resolved it now as you recommended by getting rid of the manual protocol checks. Good catch!